docs/content/preview/deploy/manual-deployment/system-config.md

```diff
@@ -23,6 +23,9 @@ In particular, the main YugabyteDB process, the YB-TServer, is multi-threaded. A
 
 ## ntp
 
+YugabyteDB is a distributed database and relies on clock synchronization to
```

Comment:
Can we  say :  YUgabyteDB relies on Clock synchronization to guarantee consistency in distributed transactions

Reply:
Done